specification/migrate/resource-manager/Microsoft.Migrate/2017-11-11-preview/migrate.json

```diff
@@ -0,0 +1,2940 @@
+          }
+        },
+        "suitabilityExplanation": {
+          "type": "string",
+          "enum": [
+            "Unknown",
+            "NotApplicable",
+            "GuestOperatingSystemArchitectureNotSupported",
+            "GuestOperatingSystemNotSupported",
+            "BootTypeNotSupported",
+            "MoreDisksThanSupported",
+            "NoSuitableVmSizeFound",
+            "OneOrMoreDisksNotSuitable",
+            "OneOrMoreAdaptersNotSuitable",
+            "InternalErrorOccuredDuringComputeEvaluation",
+            "InternalErrorOccuredDuringStorageEvaluation",
+            "InternalErrorOccuredDuringNetworkEvaluation",
+            "NoVmSizeSupportsStoragePerformance",
+            "NoVmSizeSupportsNetworkPerformance",
+            "NoVmSizeForSelectedPricingTier",
+            "NoVmSizeForSelectedAzureLocation",
+            "CheckRedHatLinuxVersion",
+            "CheckOpenSuseLinuxVersion",
+            "CheckWindowsServer2008R2Version",
+            "CheckCentOsVersion",
+            "CheckDebianLinuxVersion",
+            "CheckSuseLinuxVersion",
+            "CheckOracleLinuxVersion",
+            "CheckUbuntuLinuxVersion",
+            "CheckCoreOsLinuxVersion",
+            "WindowsServerVersionConditionallySupported",
+            "NoGuestOperatingSystemConditionallySupported",
+            "WindowsClientVersionsConditionallySupported"
+          ],
+          "description": "If machine is not ready to be migrated, this explains the reasons and mitigation steps.",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "AzureVmSuitabilityExplanation",
+            "modelAsString": true
+          }
+        },
+        "createdTimestamp": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Time when this machine was created. Date-Time represented in ISO-8601 format.",
+          "readOnly": true
+        },
+        "updatedTimestamp": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Time when this machine was last updated. Date-Time represented in ISO-8601 format.",
+          "readOnly": true
+        }
+      }
+    },
+    "ProjectKey": {
+      "type": "object",
+      "x-ms-azure-resource": true,
+      "description": "ID and Key for Migration Project.",
+      "properties": {
+        "workspaceId": {
+          "type": "string",
+          "description": "ID of Migration Project.",
+          "readOnly": true
+        },
+        "workspaceKey": {
+          "type": "string",
+          "description": "Key of Migration Project.",
+          "readOnly": true
+        }
+      }
+    },
+    "Operation": {
+      "type": "object",
+      "description": "A REST API operation supported by the provider.",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the operation.",
+          "readOnly": true
+        },
+        "display": {
```

Comment:
>display [](start = 9, length = 7)

It is better for stuff to have its own definition, instead of defining inline

Reply:
Done.